components/elm/src/biogeophys/BalanceCheckMod.F90

```diff
@@ -616,7 +616,10 @@ subroutine ColWaterBalanceCheck( bounds, num_do_smb_c, filter_do_smb_c, &
        found = .false.
        do p = bounds%begp, bounds%endp
           if (veg_pp%active(p)) then
-             if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 1.e-7_r8) ) then
+             !if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 1.e-7_r8) ) then
+             ! solar radiation balance error is high when running FATES
+             ! adjust the threshold to 5.e-7
+             if ( (errsol(p) /= spval) .and. (abs(errsol(p)) > 5.e-7_r8) ) then
```

Comment:
If we bump this value up to suppress warnings, that seems fine for a FATES run.  We are very aware that an audit needs to be done to reduce these fine errors in radiation scattering.  However, we may want to only apply it to a fates governed patch.  This "is_fates" can tell us if we are on a fates patch and if we should apply different logic: 
https://github.com/E3SM-Project/E3SM/blob/v2.0.0/components/elm/src/data_types/VegetationType.F90#L70

Reply:
I have revised the code as suggested. Thank you @rgknox @glemieux .